### ISSUE TITLE
2023.38.4 

### ISSUE THREAD

**Issue by joaquinsfy** [teslamate-org/teslamate#3386]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Update Model s plaid to 2023.38.4, car updated doesn't appear in Teslamate

### Relevant log output

```shell
2023-10-29 22:17:11.590 [error] GenStateMachine :"3" terminating
** (MatchError) no match of right hand side value: {:error, #Ecto.Changeset<action: :insert, changes: %{date: ~U[2023-10-29 22:17:11.502000Z], speed: 0, power: 0, odometer: 8852.30542, tpms_pressure_fl: Decimal.new("2.85"), tpms_pressure_fr: Decimal.new("2.85"), tpms_pressure_rl: Decimal.new("2.775"), tpms_pressure_rr: Decimal.new("2.9"), battery_heater_on: false, battery_level: 40, usable_battery_level: 39, battery_heater: false, driver_temp_setting: Decimal.new("21.0"), fan_status: 0, inside_temp: Decimal.new("22.7"), is_climate_on: false, is_front_defroster_on: false, is_rear_defroster_on: false, outside_temp: Decimal.new("20.5"), passenger_temp_setting: Decimal.new("21.0"), ideal_battery_range_km: Decimal.new("215.25"), rated_battery_range_km: Decimal.new("215.25"), est_battery_range_km: Decimal.new("182.39")}, errors: [latitude: {"can't be blank", [validation: :required]}, longitude: {"can't be blank", [validation: :required]}], data: #TeslaMate.Log.Position<>, valid?: false>}
```

### Type of installation

Manual

### Version

1.27.3

**Comment by jsight** [comment]
Yes, it is covered by this issue report: https://github.com/adriankumpf/teslamate/issues/3371

Interestingly, this is now a documented change: https://developer.tesla.com/docs/fleet-api#vehicle_data

**Comment by cicob** [comment]
I just got 2023.38.6 (EU) the day before yesterday, and Teslamate hasn't been able to fetch data since...

**Comment by guldil** [comment]
Like you @cicob i have installed yesterday 2023.38.6 (EU) and i lost access in Teslmate.

i'm using a manual installation of Teslamate.
I just had to grab the file from here https://raw.githubusercontent.com/adriankumpf/teslamate/8134150b247eec5a08572e62fdc4ebe6fda22f63/lib/tesla_api/vehicle.ex and compile.

Everything worked again.

**Comment by cicob** [comment]
Oh, ok, thanks

In the meantime I found a compiled patch over here:
https://github.com/adriankumpf/teslamate/pull/3373

I can confirm that it helped for me. The car is back online.

´´´
$cat docker-compose.yml

version: "3"

services:
  teslamate:
    #image: teslamate/teslamate:latest
    image: jsightler/teslamate:20231029
    restart: always

...
´´´

**Comment by cwanja** [comment]
This is resolved in the latest TeslaMate release 1e7c9a4
